src/preprocessing/passes/ackermann.h

```diff
@@ -38,12 +38,16 @@ namespace passes {
 using TNodeSet = std::unordered_set<TNode, TNodeHashFunction>;
 using FunctionToArgsMap =
     std::unordered_map<TNode, TNodeSet, TNodeHashFunction>;
+using USortToBVSizeMap =
+    std::unordered_map<TypeNode, size_t, TypeNode::HashFunction>;
 
 class Ackermann : public PreprocessingPass
 {
  public:
   Ackermann(PreprocessingPassContext* preprocContext);
 
+  size_t getBVSkolemSize(size_t capacity);
```

Comment:
This is a helper function like `collectUSortsToBV`, no need to declare it in the header.